### PR TITLE
fix: NFT broken card 

### LINKED ISSUE #4927: NFT card broken
image
https://beta.kodadot.xyz/bsx/gallery/3134362679-21?page=1
https://beta.kodadot.xyz/bsx/explore/items?page=1&search=Iris
this item seems to be breaking the nft cards

### PR DESCRIPTION
**Pull request by maderi21**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4927 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

**Comment by maderi21** [comment]
@yangwao I have received double payment last time (?), feel free to skip this time

File: components/shared/gallery/Gallery.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="gallery">
     <div class="is-flex is-align-self-flex-start">
       <SidebarFilter @resetPage="resetPage" />
-      <div>
+      <div class="gallery-container">
```
Comment (roiLeo) [inline components/shared/gallery/Gallery.vue:5]:
```suggestion
      <div class="container">
```

File: components/shared/gallery/Gallery.vue [hunk 2]
```diff
@@ -308,3 +308,9 @@ export default class Gallery extends mixins(
   }
 }
 </script>
+
+<style scoped>
+.gallery-container {
+  width: 100%;
+}
+</style>
```
Comment (roiLeo) [inline components/shared/gallery/Gallery.vue:316]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]
works same way if you wrap it in a `container` class

**Comment by maderi21** [comment]
@exezbcz gallery changes done

**Comment by exezbcz** [comment]
> @exezbcz gallery changes done

yes!

**Pull request merged**